src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -669,19 +631,12 @@
 helps['vm image list-offers'] = """
     type: command
     short-summary: List the VM image offers available in the Azure Marketplace.
-    examples:
-        - name: List all offers from Microsoft in the West US region.
-          text: az vm image list-offers -l westus -p Microsoft
-        - name: List all offers from OpenLocic in the West US region.
-          text: az vm image list-offers -l westus -p OpenLogic
```

Comment:
Would keep as it gives valid publisher examples. There is a typo with `OpenLocic` though.

Reply:
`OpenLogic` is the correct provider according to `az vm image list-offers -l westus`.

Reply:
The typo is in this text "List all offers from **OpenLocic**"

Reply:
Whoops, missed that. This example has still been removed and replaced, though - see the improved examples submitted in the latest commit.